shared/settings/logout/index.tsx

```diff
@@ -103,35 +105,39 @@ class LogOut extends React.Component<Props, State> {
         }}
         onClose={this.props.onCancel}
       >
-        <Kb.Box2 direction="vertical" fullHeight={true} style={styles.container}>
-          {Styles.isMobile && (
-            <Kb.Text style={styles.headerText} type="Header">
-              Do you know your password?
+        {this.state.loggingOut ? (
```

Comment:
The entire modal swiching to this is jarring - could we replace the "Just sign out" text with the spinner?

Reply:
Seems reasonable.